securedrop/source_templates/index.html

```diff
@@ -34,15 +34,16 @@
           </h2>
           <hr class="cut-out" />
           <p>If this is your first time submitting documents to journalists, start here.</p>
-          <a href="/generate" class="btn alt block" id="submit-documents-button"><img id="warning-close" src="{{ url_for('static', filename='i/font-awesome/white/cloud-upload.svg') }}" width="17px" height="17px"> Submit Documents</a>
+          {# adding a break between 'submit' and 'documents' to make both containers equal sized #}
+          <a href="/generate" class="sd-button btn alt block" id="submit-documents-button">SUBMIT<br/>DOCUMENTS</a>
```

Comment:
Check out [this link](https://docs.securedrop.org/en/latest/development/tips_and_tricks.html#using-tor-browser-with-the-development-environment) for how to allow TB to reach localhost. I was running Tor Browser 6.5 on Mac OS X when I made that screenshot. Check it out in Tor Browser (it's based on Firefox 45.7.0 so the CSS might render differently) and let me know if you can reproduce what I see above